BIPs/2024-W8/BIP-547A.json

```diff
@@ -0,0 +1,90 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1707894026213,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "Enable new rETH gauges on Arbitrum",
+    "txBuilderVersion": "1.16.3",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": ""
+  },
+  "transactions": [
+    {
+      "to": "0x5DbAd78818D4c8958EfF2d5b95b28385A22113Cd",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "name": "gauge",
+            "type": "address",
+            "internalType": "address"
+          },
+          {
+            "name": "gaugeType",
+            "type": "string",
+            "internalType": "string"
+          }
+        ],
+        "name": "addGauge",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "gauge": "0xf8A95653CC7ee59AfA2304DcC518c431a15C292C",
+        "gaugeType": "Arbitrum"
+      }
+    },
+    {
+      "to": "0x5DbAd78818D4c8958EfF2d5b95b28385A22113Cd",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "name": "gauge",
+            "type": "address",
+            "internalType": "address"
+          },
+          {
+            "name": "gaugeType",
+            "type": "string",
+            "internalType": "string"
+          }
+        ],
+        "name": "addGauge",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "gauge": "0xEaF1eef5814D9be44D0dBFb54c7773844339B7F8",
```

Comment:
this gauge is for sfrxeth/wsteth, and has nothing to do with upgrading the rate provider for wsteth/sfrxeth/reth pool `0x0c8972437a38b389ec83d1e666b69b8a4fcf8bfd`!

Reply:
Thanks for pointing out this detail which was not clearly outlined in the proposal text. We agreed to migrate to this new sfrxETH/wstETH pool without rETH in it. So I can confirm that specs of this proposal are correct and we can proceed without any changes